File: WowUp.Common/Models/GitHub/GitHubAsset.cs
using Newtonsoft.Json;
using System;

namespace WowUp.Common.Models.GitHub
{
    public class GitHubAsset
    {
        public string Url { get; set; }
        
        public long Id { get; set; }
        
        [JsonProperty("node_id")]
        public string NodeId { get; set; }
        
        public string Name { get; set; }
        
        public string Label { get; set; }

        public GitHubUploader Uploader { get; set; }

        [JsonProperty("content_type")]
        public string ContentType { get; set; }

        public string State { get; set; }

        public long Size { get; set; }

        [JsonProperty("download_count")]
        public int DownloadCount { get; set; }

        [JsonProperty("created_at")]
        public DateTime CreatedAt { get; set; }

        [JsonProperty("updated_at")]
        public DateTime UpdatedAt { get; set; }

        [JsonProperty("browser_download_url")]
        public string BrowserDownloadUrl { get; set; }
    }
}

File: wowup-electron/src/app/models/github/github-asset.ts
import { GitHubUploader } from "./github-uploader";

export interface GitHubAsset {
  url: string;
  id: number;
  node_id: string;
  name: string;
  label: string;
  uploader: GitHubUploader;
  content_type: string;
  state: string;
  size: number;
  download_count: number;
  created_at: string;
  updated_at: string;
  browser_download_url: string;
}
